src/extensions/php_fpm_metrics.go

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package extensions
+
+import (
+	"context"
+
+	agent_config "github.com/nginx/agent/sdk/v2/agent/config"
+	"github.com/nginx/agent/v2/src/core"
+	"github.com/nginx/agent/v2/src/core/config"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// Version
+	phpFpmMetricsExtensionPluginVersion = "v0.1.0"
+)
+
+type PhpFpmMetrics struct {
+	ctx       context.Context
```

Comment:
it's unusual to store a `context.Context`. Per [docs](https://pkg.go.dev/context):

> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it.

What are we planning to do with this `ctx`?

Reply:
Followed this pattern based on what agent has for their plugin, and extensions https://github.com/nginx/agent/tree/181b995a87a7c2905915f20acd0d2053c553ce64/src/extensions that collect metrics.